docs/content/manual/upgrade/4725-upgrade-shutdown-vm-in-os.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Upgrade with VM shutdown in the operating system 
```

Comment:
This is being taken care in upgrade automation test.

Reply:
In my understating, our existing upgrade backend e2e test will shutdown VM before starting upgrade.  
This manual test have a little bit difference with e2e test since in this test we shudown VM by accessing into the operating system while the e2e test shutdown VM from the virtual machine related API call. 

```
1. Login to Windows VM and shutdown machine from menu
1. Login to openSUSE VM , use command to shutdown machine
```

Thus I am thinking whether we can keep this test plan since there operations are somehow different.